index.js

```diff
@@ -8,6 +8,10 @@ const kGenerateCallbackUriParams = Symbol.for('fastify-oauth2.generate-callback-
 
 const { promisify, callbackify } = require('util')
 
+const { homepage, version } = require('./package.json')
```

Comment:
How is it ensured that we load the correct package.json? What if somebody bundles his product?

Reply:
Then the bundler should be handling all `require()` and in-line them.

Also: Is bundling an officially supported use case in the Fastify ecosystem? Is it something that is generally tested to work at large?

Reply:
This is not guaranteed.

Reply:
> This is not guaranteed.

@Uzlopak Which isn't guaranteed? That the bundler should be handling all `require()` and in-line them? Or that the Fastify ecosystem works with bundling in general?

Reply:
You expect that a bundler behaves in a specific way.

Reply:
@Uzlopak If one can't expect a bundler to actually bundle all of the stuff `require()`:ed then there's not really much one can expect about a bundler at all?

You're concerned bundlers typically wouldn't be able to resolve JSON-files that are referenced in `require()`?

If compatibility with bundlers are to be ensured then tests for one or more bundlers should really be added to ensure no dependency of `@fastify/oauth2` breaks or introduce something breaking.

Hence why I asked if bundling is an officially supported use case in the Fastify ecosystem, as then such tests should be added throughout the Fastify ecosystem.